b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
The command name is too verbose. What it really does, is that it deletes stuff, but the name starts with a "get".

Perhaps the `delete_file_version` command should have the `<fileName>` as an optional argument instead of creating another command? It should then be documented that passing the `fileName` argument has strong positive performance impact, so it should be provided whenever possible.